libraries/bot-azure/src/main/java/com/microsoft/bot/azure/CosmosDbPartitionedStorageOptions.java

```diff
@@ -31,6 +31,7 @@ public CosmosDbPartitionedStorageOptions() {
         connectionPolicy = ConnectionPolicy.GetDefault();
         consistencyLevel = DEFAULT_CONSISTENCY;
         containerThroughput = DEFAULT_THROUGHPUT;
+        compatibilityMode = true;
```

Comment:
I'm not quite sure if this is the correct place. Check [JS](https://github.com/microsoft/botbuilder-js/blob/main/libraries/botbuilder-azure/src/cosmosDbPartitionedStorage.ts#L154)